model/trace/instrumentation/instrumentation.yml

```diff
@@ -0,0 +1,13 @@
+groups:
+- id: instrumentation.scope
+  type: scope
+  brief: >
+    The `instrumentation` conventions are intended to provide common
+    mechanisms for triangulating and identifying instrumentation libraries.
+  prefix: instrumentation
+  stability: experimental
+  attributes:
+  - ref: instrumentation.source.repo
+    requirement_level: opt_in
```

Comment:
does it need to be opt-in? I.e. users would need to enable it explicitly. Why not `recommended: if not captured otherwise, e.g. in the instrumentation scope name attribute`? (i.e. instrumentation libraries may decide to populate it if it makes sense)

Reply:
I did not realize earlier that `recommended` could be conditional. I followed that pattern in https://github.com/open-telemetry/semantic-conventions/pull/1094